docs/dev/compiler-toolchain/overview.md

```diff
@@ -0,0 +1,68 @@
+# Overview
+
+This section introduces an LLVM-based compiler toolchain for smart contract languages with Ethereum Virtual Machine
+(EVM) support. The toolchain works on top of existing compilers and requires their output, which typically includes
+intermediate representations (IRs), abstract syntax trees (ASTs), and auxiliary contract metadata and documentation.
+::: info
+At the time of writing, we support Solidity and Vyper.
+:::
+
+The toolchain consists of the following:
+
```

Comment:
I’d suggest including the diagram from our Onboarding wiki here.

Reply:
Added the image!
@kailash-manasarovar please check if it's consistent with others.

Reply:
i thought i'd resolved this one, we can check on staging when committed